docs/.vuepress/components/ListCard.vue

```diff
@@ -108,20 +110,16 @@ export default {
       align-items center
       grid-column 3
       content '↗️'
+      grid-row: 1/999
```

Comment:
No `:` needed. What's the `1/999` do in effect? I tried toggling this in the browser & didn't notice a difference.

Reply:
1/999 ensures that the content spans all available rows. Really, we could do 1/3 here, as it has a max of 3 rows, but 1/999 keeps things simple and will never have to be updated. It's sort of like setting `z-index` to `9999`